### PR TITLE
Changes needed for dependency upgrade qt 5.12

### PR DESCRIPTION
**Pull request by k-dominik**
qt 5.12 in combination with Python >= 3.7.4 will bubble up some errors that would previously be ignored (see also https://github.com/ilastik/volumina/pull/261).

This PR includes similar changes to main ilastik.

Todo before this can be merged:

- [x] rebuild on win
- [x] rebuild on osx
  - [x] test on osx
- [x] move packages to main channel
- [x] remove the `REMOVE BEFORE MERGE` commits ;)

### CODE REVIEW

File: ilastik/applets/dataSelection/dataSelectionGui.py [hunk 1]
```diff
@@ -601,9 +600,9 @@ def addStack(self, roleIndex, laneIndex):
         try:
             # FIXME: do this inside a Request
             self.parentApplet.busy = True
-            inner_path = stack_info.importAsLocalDataset(project_file=self.project_file)
+            inner_path = stack_info.importAsLocalDataset(project_file=self.get_project_file())
```
Comment (m-novikov) [inline ilastik/applets/dataSelection/dataSelectionGui.py:603]: Does new qt handle properties in a special way?

**Review by m-novikov — APPROVED** [review]

File: ilastik/applets/dataSelection/dataSelectionGui.py (hunk 1 above)
Comment (k-dominik) [reply ilastik/applets/dataSelection/dataSelectionGui.py:603]: yes

> The cause is PyQt5 accessing all class attributes via PyObject_GetAttr during initialization of the Gui Widget.
In our case, the class is not fully initialized by that time but class properties are still executed (via PyObject_GetAttr).

File: ilastik/applets/counting/countingGuiBoxesInterface.py [hunk 2]
```diff
@@ -183,7 +183,7 @@ class QGraphicsResizableRect(QGraphicsRectItem):
     hoverColor = QColor(255, 0, 0)  # _hovering and selection color
 
     def __init__(self, x, y, h, w, scene=None, parent=None, editor=None):
-        """"
+        """ "
```
Comment (m-novikov) [inline ilastik/applets/counting/countingGuiBoxesInterface.py:186]: typo

Comment (k-dominik) [reply ilastik/applets/counting/countingGuiBoxesInterface.py]: thx, removed that!

**Pull request merged**